x-pack/solutions/observability/plugins/observability_onboarding/ui_tests/fixtures/index.ts

```diff
@@ -32,6 +39,18 @@ export const test = testFixtures.extend<ExtendedScoutTestFixtures, ExtendedScout
 
     await use(extendedPageObjects);
   },
+  apiServices: [
+    async (
+      { apiServices, kbnClient }: { apiServices: ObltApiServicesFixture; kbnClient: KbnClient },
+      use: (extendedApiServices: ExtendedApiServicesFixture) => Promise<void>
+    ) => {
+      const extendedApiServices = apiServices as ExtendedApiServicesFixture;
+      extendedApiServices.onboarding = getOnboardingApiHelper(kbnClient);
```

Comment:
Should we log somewhere that the onboarding API service was loaded?

We deleted the file `onboarding_api.ts` which included

```ts
log.serviceLoaded('onboardingApi');
```

Perhaps we may want to add that back in.

Reply:
oh I missread it. With this PR we don't start individual services for APIs, but a single one `apiServices`. The message for fixture start is logged and inherited in in this extension.

Reply:
Thanks for chiming in here. What is it exactly that you misread when you say "I missread it"?

Reply:
Initially I agreed with your idea to log `onboardingApi`, this is what we do for fixtures in order to catch the ones loaded too often (test vs worker scope). But since onboarding is now part of unified `apiServices` fixture I don't see a need to log individual message, we only log when `apiServices` is loaded first itself

Reply:
That makes sense to me! Thank you for clarifying.